### PR TITLE
Update the README

### PR DESCRIPTION
**Pull request by rcedwards**
* installation details
* usage examples
* stack design methodology
* Big Nerd Ranch info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,265 @@
...
 ![Legend](./Resources/Legend.png)
 
-### Nested Managed Object Context Stack Type
+### <a id="shared_coordinator"></a> Shared Persistent Store Coordinator Pattern
 
-![Nested Managed Object Context Stack](./Resources/NestedMOC.png)
+The first pattern is constructed with a single persistent store coordinator, connected to a main queue managed object context. Background worker managed object contexts are created as needed and share the same single instance of the persistent store coordinator. 
 
-### Shared Persistent Store Coordinator Stack Type
+Saving any one of these managed object contexts will push changes through the shared coordinator and on to the backing store file. While the changes from each context are available in the store after a save, no other context is aware of the changes made in any other context.
```
Comment (stevesparks) [inline README.md]: This reads strangely... strike the first 'other'? 

`no context is aware of the changes made in any other context.`

File: README.md [hunk 2]
```diff
@@ -1,40 +1,265 @@
-# BNR CoreData Stack - README is a Work in Progress
```
Comment (lyricsboy) [inline README.md]: s/CoreData/Core Data/g when referring to the technology. That's how Apple says it.

**Comment by lyricsboy** [comment]
I'd be inclined to extract the lovely descriptions of all the stacks with the diagrams into a blog post and keep the README short and sweet. You can link to the blog post from the README once it's published.

**Comment by rcedwards** [comment]
@lyricsboy @stevesparks updated with suggestions.

**Comment by stevesparks** [comment]
:+1:

**Pull request merged**